js/validation-form.js

```diff
@@ -0,0 +1,41 @@
+const quantityRoomsForm = document.querySelector('#room_number');
+const quantityGuestsPlaceForm = document.querySelector('#capacity');
+const quantityGuestValues = quantityGuestsPlaceForm.children;
+
+
+const disablePlaceForm = function (listElements) {
+  for (let currentValue of listElements) {
+    currentValue.setAttribute('disabled', 'disabled');
+  }
+};
+
+const checkRoomsPlaceForm = function () {
+  disablePlaceForm(quantityGuestValues);
+  if (this.value === '1') {
+    quantityGuestsPlaceForm.options[2].removeAttribute('disabled');
+    quantityGuestsPlaceForm.value = '1';
+  } else if (this.value === '2' && quantityGuestsPlaceForm.value === '1' || this.value === '2' && quantityGuestsPlaceForm.value === '2') {
+    quantityGuestsPlaceForm.options[1].removeAttribute('disabled');
+    quantityGuestsPlaceForm.options[2].removeAttribute('disabled');
+  } else if (this.value === '2') {
+    quantityGuestsPlaceForm.options[1].removeAttribute('disabled');
+    quantityGuestsPlaceForm.options[2].removeAttribute('disabled');
+    quantityGuestsPlaceForm.value = '2';
```

Comment:
Все работает, но лучше не повторять код а записать например так
```suggestion
  } else if (this.value === '2') {
    quantityGuestsPlaceForm.options[1].removeAttribute('disabled');
    quantityGuestsPlaceForm.options[2].removeAttribute('disabled');
    if (quantityGuestsPlaceForm.value === '0' || quantityGuestsPlaceForm.value === '3') {
      quantityGuestsPlaceForm.value = '2';
    }
  } 
```
и так далее